src/emqtt_ws.erl

```diff
@@ -40,11 +40,11 @@ connect(Host0, Port, Opts, Timeout) ->
     {ok, _} = application:ensure_all_started(gun),
     %% 1. open connection
     TransportOptions = proplists:get_value(ws_transport_options, Opts, []),
-    ConnOpts = #{connect_timeout => Timeout,
+    TransportOpts = maps:from_list(TransportOptions),
```

Comment:
it should be `TransportOpts = #{transport_opts => TransportOptions},` ?
I tried to fix it here, and verified in CI.
https://github.com/emqx/emqtt/pull/218/files#diff-8ec1346450794b68c5e1451510579efbbe75e28c4ef218329434a128402fcd09R43